src/update_xs_yum.ml

```diff
@@ -527,6 +527,13 @@ let _ =
         (artifactory // branch // n ) "source-retail.iso"
         s3bucket) >>= fun () ->
 
+    let branch = "feature/vgpu-migration/emu-manager" in
```

Comment:
@robhoes I thought we needed the `feature/vgpu-migration/master` branch, but maybe I remember incorrectly?